docusaurus/docs-internals/editor-controls/data-controls/toggle.md

```diff
@@ -0,0 +1,177 @@
+---
+sidebar_position: 2
+toc_max_heading_level: 4
+---
+
+# Toggle
+
+The toggle control in Brizy works as a switchable interactive element, allowing users to switch between multiple states.
+
+Example of the toggle:
+
+![Toggle](/img/data-controls/toggle-left.png)<br/>
+![Toggle](/img/data-controls/toggle-center.png)<br/>
+![Toggle](/img/data-controls/toggle-right.png)<br/>
+
+### Parameters
+
+| Name        | Type                                     | Default | Description                                                                                                                                                                                                                                                                                                                                                                                               |
+| :---------- | :--------------------------------------- | :-----: | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`        | `string`                                 |    -    | The identifier of the key where the switch will save your data                                                                                                                                                                                                                                                                                                                                            |
+| `type`      | `string`                                 |    -    | Type should be `"switch"` to use this control                                                                                                                                                                                                                                                                                                                                                             |
+| `label?`    | `string`                                 |    -    | The label displayed on the left side of the control                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
this options can't work inside popover items, demo: https://jmp.sh/Fhu2NZSg

remove label and for future check if your option look good with label